webapp/lib/controller.dart

```diff
@@ -821,17 +822,14 @@ void command(UIAction action, Data data) {
         view.snackbarView.hideSnackbar();
       }
       // If the shortcut is for a reply, find it and send it
-      var selectedReply = suggestedRepliesByCategory[selectedSuggestedRepliesCategory].where((reply) => reply.shortcut == keyPressData.key);
+      var selectedReply = suggestedRepliesByCategory[selectedSuggestedRepliesCategory].where((reply) => reply.shortcut == keyPressData.key && !keyPressData.hasModifierKey);
```

Comment:
Rather than this change, perhaps wrap lines 825 - 834 in
```
if (!keyPressData.hasModifierKey) {
    ...
}
```
so that the "hasModifierKey prevents all replies" is more obvious?

Reply:
Thanks @danrubel that's indeed more straightforward. On second thought, we also want to ignore modifier key-shortcuts for tags as well, so I replaced this with an early exit if a modifier key is present. PTAL

Reply:
@marianamarasoiu Good change. I like that even better.